packages/experiments/src/components/Coachmark/Coachmark.Props.ts

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Coachmark } from './Coachmark';
+import { ICoachmarkStyles, ICoachmarkStyleProps } from './Coachmark.Styles';
+import { IPositioningContainerProps } from '../PositioningContainer/PositioningContainer.Props';
+import {
+  IPoint
+} from '../../Utilities';
+export type IStyleFunction<TStylesProps, TStyles> = (props: TStylesProps) => Partial<TStyles>;
+
+export interface ICoachmark {
+}
+
+export interface ICoachmarkProps extends React.Props<Coachmark> {
+  /**
+  * All props for your component are to be defined here.
+  */
+  componentRef?: (component: ICoachmark) => void;
+
+  /**
+   * Get styles method
+   */
+  getStyles?: IStyleFunction<ICoachmarkStyleProps, ICoachmarkStyles>;
+
+  /**
+   * The target that the TeachingBubble should try to position itself based on.
+   * It can be either an HTMLElement a querySelector string of a valid HTMLElement
+   * or a MouseEvent. If MouseEvent is given then the origin point of the event will be used.
+   */
+  positioningTarget?: HTMLElement | string | MouseEvent | IPoint | null;
```

Comment:
i would rename this to "target" for consistency with other things.

Reply:
or... rename everything else? but it's better to just be consistent and predictable.